src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -300,22 +300,28 @@ def __call__(self, input_ids, attention_mask, token_type_ids, position_ids):
         return encoder, pooled
 
 
+class FlaxBertPretrainedModel(FlaxPreTrainedModel):
+    """
+    An abstract class to handle weights initialization and a simple interface for downloading and loading pretrained
+    models.
+    """
+
+    config_class = BertConfig
+    base_model_prefix = "bert"
+
+
 @add_start_docstrings(
     "The bare Bert Model transformer outputting raw hidden-states without any specific head on top.",
     BERT_START_DOCSTRING,
 )
-class FlaxBertModel(FlaxPreTrainedModel):
+class FlaxBertModel(FlaxBertPretrainedModel):
     """
     The model can behave as an encoder (with only self-attention) as well as a decoder, in which case a layer of
     cross-attention is added between the self-attention layers, following the architecture described in `Attention is
     all you need <https://arxiv.org/abs/1706.03762>`__ by Ashish Vaswani, Noam Shazeer, Niki Parmar, Jakob Uszkoreit,
     Llion Jones, Aidan N. Gomez, Lukasz Kaiser and Illia Polosukhin.
     """
 
-    model_class = FlaxBertModule
```

Comment:
I don't really think we need this class attribute no? The loading seems to work fine without it and it would add some more complexity to the model design.

Reply:
Mostly relica of the time I started the Flax impl. Works for me removing it